internal/bminventory/inventory.go

```diff
@@ -1482,6 +1483,22 @@ func (b *bareMetalInventory) GetClusterInstallConfig(ctx context.Context, params
 	return installer.NewGetClusterInstallConfigOK().WithPayload(string(cfg))
 }
 
+func (b *bareMetalInventory) GetClusterDefaultConfig(ctx context.Context, params installer.GetClusterDefaultConfigParams) middleware.Responder {
+	var c Config
+	body := models.ClusterDefaultConfig{}
+
+	if err := envconfig.Process("default-conf", &c); err != nil {
+		return common.GenerateErrorResponder(err)
+	}
+
+	body.NtpSource = c.DefaultNTPSource
+	body.ClusterNetworkCidr = DefaultClusterNetworkCidr
```

Comment:
I think that those consts need to move the Config as well. 
If we want to change those default we will not need to have a new release.